pieman.sh

```diff
@@ -204,7 +206,7 @@ check_required_directories
 check_required_files
 
 info "checking toolset"
-. toolset.sh
+. ./toolset.sh
```

Comment:
The change is useless.

Reply:
It was an attempt to meet linter requirements. I will back it as was.